src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -428,4 +438,122 @@ public synchronized File getNextMixinMappings() {
 	public List<LoomDecompiler> getDecompilers() {
 		return decompilers;
 	}
+
+	public void runs(Closure<?> conf) {
+		runs.configure(conf);
+	}
+
+	// Workaround for ugly closures in Kotlin DSL buildscripts
+	public void run(String name, Action<RunConfigSettings> settingsAction) {
+		runs.create(name, settingsAction);
+	}
+
+	public NamedDomainObjectContainer<RunConfigSettings> getRuns() {
+		return runs;
+	}
+
+	public static class RunConfigSettings implements Named {
```

Comment:
This class should go into its own file, this class is getting quite large these days.